{{cookiecutter.github_repository}}/provisioner/roles/project_data/defaults/main.yml

```diff
@@ -5,6 +5,15 @@ pg_user: dev
 pg_password: password
 django_requirements_file: requirements.txt
 
+# asgi related variables
+asgi_user: www-data
+asgi_group: www-data
+asgi_workers: 2
+asgi_socket: /tmp/django-{{ domain_name }}-asgi.sock
+asgi_user: www-data
+asgi_group: www-data
+asgi_workers: 2
+
```

Comment:
This should be surrounded by if-else clause for `add_asgi`

Reply:
Done